source/modules/lesson00/expectations.rst

```diff
@@ -5,26 +5,26 @@ Expectations
 What does success on this class look like?
 ==========================================
 
-For classroom the usual 80% attendance is required. Contact your instructor as
-soon as possible if you have any special circumstances.
+For classroom, the usual 80% attendance is required. Contact your instructor as
+soon as possible if you have any special circumstances that will prevent you from meeting this requirement.
 
 For all class delivery methods you will need to achieve a pass rate of 80%
 for the class assignments and quizzes. See the grading rubric for more details.
 
 .. _grading-rubric:
     grading
 
-Also, there are usually ungraded activities for the lessons. This is work
-that you can do to help you really understand and apply the concepts. Although
-the work is ungraded, we expect that you will want to do the work!
+Also, most lessons include ungraded activities. This is work
+that you can do to help you improved your understanding and apply the concepts learned. Although
+ungraded, we expect that you will want to do the work!
 
-Since giving demos is an important part of a modern develops toolset, your classroom
+Since giving demos is an important part of a modern devops toolset, your classroom
```

Comment:
developer's ?

Reply:
Makes sense, I'll change it to "developer's".